ls1intum/Artemis
# [Feature] Collapsible List Items For Feedback

TODO:
### Checklist

### Motivation and Context
 Feedback items have long text making the feedback view crowded
 #3217

### Description
 Created feedback-collapse.component which takes the feedbacks text and makes it un-/collapsible if it has more than 200 characters.

### Steps for Testing

Easiest way i found is this:
0. Be logged in as instructor
1. Navigate: Course Management -> Some Course -> Exercises -> If there are none generate a basic one for testing -> Edit in Editor 
2. Now you should be able to see the feedback on the top right
3. if none of the feedback messages are over 200 characters go in the test repository and change a message to be longer 
![testing_feedback](https://user-images.githubusercontent.com/56715566/115446182-18dcc080-a217-11eb-8fad-21e3a2b1a2fa.gif)
4. Save and submit
5. look at feedback


### Test Coverage




TODO

### Screenshots


![feedback_collapse](https://user-images.githubusercontent.com/56715566/115445061-aae3c980-a215-11eb-9dcb-bc4b7bbed099.gif)
